samples/language/TypeDeclarations.qs

```diff
@@ -2,33 +2,25 @@
 /// Type Declarations
 ///
 /// # Description
-/// User-defined types, or UDTs as they are commonly called, are supported
-/// in Q#. They are immutable but support a copy-and-update construct.
+/// User-defined struct types are supported in Q#.
+// They are immutable but support a copy-constructor syntax.
```

Comment:
Dropped a `/` here, making it a double instead of triple, so it's not a doc comment.

Reply:
Good catch! Fixed